.storybook/main.js

```diff
@@ -1,52 +1,37 @@
 const path = require('path');
 
 module.exports = {
-  stories: ['../src/**/*.stories.tsx'],
-  addons: [
-    '@storybook/addon-essentials',
-    '@storybook/addon-links',
-    '@storybook/addon-storysource',
-    '@storybook/addon-a11y/register',
+  "stories": [
+    "../src/**/*.stories.tsx"
+  ],
+  "addons": [{
+      name: '@storybook/preset-scss',
+      options: {
+        cssLoaderOptions: {
+          modules: {
+            auto: true
+          }
+        }
+      }
+    },
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-a11y"
```

Comment:
Did you remove this part on purpose?

```
 '@storybook/addon-storysource',
 '@storybook/addon-a11y/register',
```

Reply:
`@storybook/addon-a11y` was set-up in `preview.js` which is not the [right way](https://www.npmjs.com/package/@storybook/addon-a11y) to set it up along with using `@storybook/addon-a11y/register`, so removed the usage from `preview.js` and added the correct syntax here.